### ISSUE TITLE
libcoveweb 0.30.5 was somehow released with a modified input.html

### ISSUE THREAD

**Issue by jpmckinney** [OpenDataServices/lib-cove-web#147]
This is missing (though it's in git and there is no commit that removes it):

https://files.pythonhosted.org/packages/18/35/4db3946fdb4e073408c97af2211773d78ccb80111568019705fb69385312/libcoveweb-0.30.5.tar.gz

```jinja
{% block extrafooterscript %}
 <script>
   //If javascript is off all pannels show.
   //We use this to collapse pannels 2 and 3 if javascript is on.
  $("#collapseTwo").attr('class', 'panel-collapse collapse');
  $("#collapseThree").attr('class', 'panel-collapse collapse');

  $('form').submit(function() {
      $('#loading').css({"display": "block"});
      $('#accordion').hide();
  });
  $(window).unload(function() {
    $('#loading').css({"display": "none"});
  })

</script>
{% endblock %}
```

**Comment by jpmckinney** [comment]
Since there is no workflow for PyPI, I assume whoever released it did so with a dirty local repository.

Please release using a workflow to avoid user error: https://ocp-software-handbook.readthedocs.io/en/latest/python/packages.html#release-process

**Comment by michaelwood** [comment]
Ack sorry about that. I've been removing jquery dependency in 360 Cove and was testing something, didn't realise the dist wouldn't clean before creating the tarball

**Comment by ghost** [comment]
I've released 0.31.0 and yanked 0.30.5

It is a manual process.

In the short term I've improved our documentation to make clearer the step about checking it's clean.

In the long term I personally would love to move to a GitHub workflow, but I don't know when we'll get to that. I've noted the OCP docs to look at when we do that. I'll close this issue tho, as we'll do this internally on a co-op board on all our repositories.
